### DISCUSSION TITLE
Time tracker

### DISCUSSION THREAD

**Discussion by coolscribble** [RetroAchievements/RAWeb#1241] [Ideas]
Hello,
I don't know how hard it could be, but it would be nice if game pages had 2 new things on them (can be even more) related to time "required to master".
My idea would be take data which is shown in RATools and make an average out of them and put them as: Average time to master. (maybe with deleting those which are 0h because they will break the time limit) also I would like if there was written how much time did you spent on playing/mastering it (if it's no possible to show how long you played the game already doesn't matter just mastery is enough).

**Comment by Jamiras** [comment]
I think this is covered by #714
